carbonmark-data/app/[locale]/details/token-volume-over-time/[bridge]/page.tsx

```diff
@@ -0,0 +1,15 @@
+import { t } from "@lingui/macro";
+import DetailPage from "components/pages/DetailPage";
+import useBridgeInfo from "hooks/useBridgeInfo";
+
+export default function TokenVolumeOverTimePage() {
+  const { bridgeLabel } = useBridgeInfo();
+  return (
+    <DetailPage
+      pageTitle={t`${bridgeLabel} Volume Over Time<`}
+      chartTitle={t`${bridgeLabel} Volume Over Time<`}
```

Comment:
```suggestion
      pageTitle={t`${bridgeLabel} Volume Over Time`}
      chartTitle={t`${bridgeLabel} Volume Over Time`}
```

this looks like a typo